libzeth/core/extended_proof.hpp

```diff
@@ -2,36 +2,35 @@
 //
 // SPDX-License-Identifier: LGPL-3.0+
 
-#ifndef __ZETH_EXTENDED_PROOF_HPP__
-#define __ZETH_EXTENDED_PROOF_HPP__
+#ifndef __ZETH_TYPES_EXTENDED_PROOF_HPP__
```

Comment:
`__ZETH_TYPES_EXTENDED_PROOF_HPP__` this does not seem consistent with the rest. I think it should be `__ZETH_CORE_EXTENDED_PROOF_HPP__`